src/cortxfs/cortxfs_ops.c

```diff
@@ -79,19 +79,23 @@ static int __cfs_getattr(struct cfs_fs *cfs_fs, const cfs_cred_t *cred,
 {
 	int rc;
 	struct stat *stat = NULL;
-	struct kvstore *kvstor = kvstore_get();
-	struct kvnode node = KVNODE_INIT_EMTPY;
+	struct cfs_fh *fh = NULL;
 
-	dassert(cred != NULL);
-	dassert(ino != NULL);
-	dassert(kvstor != NULL);
+	dassert(cfs_fs && cred && ino && bufstat);
+
+	/* TODO: Should get rid of creating and destroying FH operation in this
```

Comment:
Any suggestion on a unique tag I should be putting along with this comment and everywhere from now onwards when I have to create a temporary FH until caller pass to the API so that at the end grepping it will help to determine we haven't missed to remove them anywhere or we aren't missing to pass FH from caller to some APIs.
```
Something like : /* TODO: CFSFH Comment */ 
```

Reply:
/* Temp_FH_op - to be removed */